Makefile

```diff
@@ -17,7 +17,7 @@ down: ## stop and remove learner-portal container
 	docker-compose down
 
 npm-install-%: ## install specified % npm package on the learner-portal container
-	docker exec npm install $* --save-dev
+	docker exec frontend-app-learner-portal_web_1 npm install $* --save-dev
```

Comment:
Should probably give the container_name in the docker-compose.yml rather than assume the name here. Also seems unrelated to the above change so maybe just push a separate PR for the Makefile/docker-compose changes

Reply:
Whoops, I forgot I did this. It's actually also in @tuchfarber's pr. I'm going to remove it.

Reply:
oops. That wasn't supposed to make it into my PR. That was supposed to be a temporary quick fix.